### DISCUSSION TITLE
Final Submission 💯

### DISCUSSION THREAD

**Discussion by khoa-l** [hack4impact-calpoly/starter-pack#5] [Submissions]
Submit your completed website here!

No changes after your submission will be reviewed. Link your website _and_ the repository.
Great work! :D

**Comment by TeachMeTW** [comment]
[Website](https://hack4impact-starter.vercel.app/index.html)
[The Repo/Sauce](https://github.com/TeachMeTW/hack4impact_starter)

**Reply by artiwari07** [reply]
website: https://artiwari07.github.io/
repo: https://github.com/artiwari07/artiwari07.github.io.git

**Comment by munozAndrew** [comment]
website: https://munozandrew.github.io/PortfolIoHACK4IMPACT/
repo: https://github.com/munozAndrew/PortfolIoHACK4IMPACT

**Reply by Robert303V** [reply]
10/10 images

**Comment by Sohini-pillay** [comment]
website: [https://sohini-pillay.github.io/](https://sohini-pillay.github.io/)
repo: [https://github.com/Sohini-pillay/Sohini-pillay.github.io](https://github.com/Sohini-pillay/Sohini-pillay.github.io)

**Comment by aarav27** [comment]
Website: https://aarav27.github.io/
Repository: https://github.com/aarav27/aarav27.github.io

**Reply by khoa-l** [reply]
Honestly, a bop

**Reply by Robert303V** [reply]
Wow, well played my friend.

**Reply by reakunen** [reply]
AAAARAAAAV THE GOAAAT!!!!!

**Comment by charlesmoreno32** [comment]
Website: [https://charlesmoreno32.github.io/](https://charlesmoreno32.github.io/)
Repository: [https://github.com/charlesmoreno32/charlesmoreno32.github.io](https://github.com/charlesmoreno32/charlesmoreno32.github.io)

**Comment by khoawack** [comment]
[Website](https://khoawack.com/khoa.github.io/)
[Repo](https://github.com/khoawack/khoa.github.io)

**Comment by sapierre26** [comment]
Website: [sapierre26.github.io](https://github.com/sapierre26/sapierre26.github.io)
Repo: https://github.com/sapierre26/sapierre26.github.io.git

**Reply by khoa-l** [reply]
Hi Sanaia, I think your repository should be `sapierre26.github.io`. All you'd need to do is change the name of the repository! You can do this in the settings of your repository!

<img width="1440" alt="Screenshot 2023-09-22 at 01 22 07" src="https://github.com/hack4impact-calpoly/starter-pack/assets/74576123/c1a805b4-67a8-4ab3-a2d0-286761f16c96">

This would let you automatically host the page. Would you please resubmit with the correct links? Let us know if you need any help as you may have to reconnect your VSCode to your renamed repository!

In which case you would run this command `git remote set-url origin NEW_URL`.

**Comment by RyanSchmit** [comment]
Website: https://ryanschmit.github.io/
Repo: https://github.com/RyanSchmit/RyanSchmit.github.io

**Comment by devinhadley** [comment]
[Website](https://devinhadley.github.io)

[Repo](https://github.com/devinhadley/devinhadley.github.io)

**Comment by dakpasala** [comment]
Website: https://dakpasala.github.io/
Repository: https://github.com/dakpasala/dakpasala.github.io

**Comment by ruiyichi** [comment]
Website: https://ruiyichi.github.io/
Repository: https://github.com/ruiyichi/ruiyichi.github.io

**Reply by TeachMeTW** [reply]
I really like the embedded PDF viewer

**Comment by wbrandon25** [comment]
Website: https://wbrandon25.github.io/
Repository: https://github.com/wbrandon25/wbrandon25.github.io

**Comment by Virnirr** [comment]
Website: https://virnirr.github.io/zhi.github.io/
Repository: https://github.com/Virnirr/zhi.github.io

**Comment by Bdon101** [comment]
website: https://bdon101.github.io
repo: https://github.com/Bdon101/bdon101.github.io

**Reply by Robert303V** [reply]
I love the stylings of your site!

**Comment by evayktv** [comment]
[website](https://evayktv.github.io/index.html)
[repo](https://github.com/evayktv/evayktv.github.io)

**Comment by Sriya-Adunur** [comment]
Website: https://sriya-adunur.github.io/index.html
Repo: https://github.com/Sriya-Adunur/Sriya-Adunur.github.io.git

**Reply by javalosr2004** [reply]
Hi Sriya, I was looking at your website, and I recommend that the navbar should remain on the top of the website for every page!

**Comment by Robert303V** [comment]
Website: https://r303vermeulen.github.io/
Repo: https://github.com/R303Vermeulen/R303Vermeulen.github.io

Hope you enjoy :)

**Reply by lukewaltz** [reply]
Wow I love it, Robert!

**Comment by ethantrantalis** [comment]
Here is a link to my [website](https://rob0t3000.github.io/).
Here is my GitHub [repository](https://github.com/Rob0t3000/Rob0t3000.github.io).
Thank you for making such a thorough and helpful starter pack, I will surely refer to it again later.

**Comment by tomaseuu** [comment]
Website: https://tomaseuu.github.io/
Repository: https://github.com/tomaseuu/tomaseuu.github.io

Thank you again for the starter pack, I definitely learned a lot about html and css!

**Comment by Brian-Kwong** [comment]
Webpage : https://brian-kwong.github.io
Repo : https://github.com/Brian-Kwong/Brian-Kwong.github.io/tree/main

**Comment by HoldenEv** [comment]
webpage: https://holdenev.github.io/
Repository: https://github.com/HoldenEv/HoldenEv.github.io

**Comment by SilveerDusk** [comment]
Website: [https://silveerdusk.github.io/](https://silveerdusk.github.io/)
Repo: [https://github.com/SilveerDusk/SilveerDusk.github.io](https://github.com/SilveerDusk/SilveerDusk.github.io)

**Comment by jqueja** [comment]
Website: https://jqueja.github.io/
Repo: https://github.com/jqueja/jqueja.github.io

**Comment by arnavbhola123** [comment]
website: https://arnavbhola123.github.io/
repo: https://github.com/arnavbhola123/arnavbhola123.github.io

**Comment by lukewaltz** [comment]
Website: https://lukewaltz.github.io/
Repository: https://github.com/lukewaltz/lukewaltz.github.io
This was very fun and satisfying to make! I intend to continue to contribute to my website as my HTML and CSS skills grow.

**Reply by Robert303V** [reply]
The site looks great, I love that you used the blog! Also great work with the landing page for your internship.

**Comment by davism7** [comment]
Website: https://davism7.github.io/
Repo: https://github.com/davism7/davism7.github.io.git

**Comment by Jhammett03** [comment]
Website: [Website](https://jhammett03.github.io/)
Repo: [Repo](https://github.com/Jhammett03/Jhammett03.github.io)

**Comment by aelamanc** [comment]
Website: https://aelamanc.github.io/
Repo: https://github.com/aelamanc/aelamanc.github.io

**Comment by amandayu255** [comment]
Website: https://amandayu255.github.io/ 
Repo: https://github.com/amandayu255/amandayu255.github.io

**Comment by Kaweees** [comment]
Website: https://miguelvf.dev/
Repository: https://github.com/Kaweees/kaweees.github.io

**Comment by JoaquinArredondo** [comment]
website: https://joaquinarredondo.github.io/
Repository: https://github.com/JoaquinArredondo/JoaquinArredondo.github.io.git

**Comment by Mish0322** [comment]
website: https://mish0322.github.io/
repo: https://github.com/Mish0322/Mish0322.github.io

**Comment by riannalei** [comment]
website: [https://riannalei.github.io/index.html](https://riannalei.github.io/index.html)
repo: [https://github.com/riannalei/riannalei.github.io.git](https://github.com/riannalei/riannalei.github.io.git)

**Comment by lab596** [comment]
[Website](https://lab596.github.io/index.html)
[Repo](https://github.com/lab596/lab596.github.io)

**Comment by noahgiboney** [comment]
Website: https://noahgiboney.github.io/
Repository: https://github.com/noahgiboney/noahgiboney.github.io

**Comment by cbgabler** [comment]
Website: https://cbgabler.github.io/index.html
Repo: https://github.com/cbgabler/cbgabler.github.io

**Comment by arun-codes-here** [comment]
Website link: https://arun-codes-here.github.io/
Repo: https://github.com/arun-codes-here/arun-codes-here.github.io

**Comment by NoahMoscovici** [comment]
Website: [https://noahmoscovici.github.io/](https://noahmoscovici.github.io/)
Repo: [https://github.com/NoahMoscovici/noahmoscovici.github.io](https://github.com/NoahMoscovici/noahmoscovici.github.io)

**Comment by wesleytam88** [comment]
[Website](https://wesleytam88.github.io/)
[Repo](https://github.com/wesleytam88/wesleytam88.github.io)

**Comment by AceOfRay** [comment]
Website: https://aceofray.github.io/AceOfRaysPersonalWebsite/index.html
Repo: https://github.com/AceOfRay/AceOfRay.github.io.git

**Comment by nluong9600** [comment]
Website: https://nluong9600.github.io/
Repo: https://github.com/nluong9600/nluong9600.github.io

**Comment by logancostello** [comment]
[Website](https://logancostello.github.io/rl)
[Repo](https://github.com/logancostello/logancostello.github.io)

Thank you for the wonderful starter pack!

**Comment by jmdebarro** [comment]
Final Submission!

[Website](https://jmdebarro.github.io/)
[Repo](https://github.com/jmdebarro/jmdebarro.github.io)

**Reply by reakunen** [reply]
ohmy GASH jonathan debarro

**Reply by jmdebarro** [reply]
Brian, tuck in your shirt

**Comment by greinhard2003** [comment]
Website: https://greinhard2003.github.io/
Repo: https://github.com/greinhard2003/greinhard2003.github.io

**Comment by achou05** [comment]
Website: https://achou05.github.io/
Repo: https://github.com/achou05/achou05.github.io

Thank you for this amazing experience!

**Comment by jacksonmcl** [comment]
Website: https://jacksonmcl.github.io/
Repo: https://github.com/jacksonmcl/jacksonmcl.github.io

**Comment by Nathaynn** [comment]
[Website](https://nathaynn.github.io)
[Repo](https://github.com/Nathaynn/Nathaynn.github.io)
This was pretty fun and I learned a ton. Thanks :]

**Comment by mtbosio** [comment]
Website: https://mtbosio.github.io/
Repo: https://github.com/mtbosio/mtbosio.github.io

**Comment by Elmanhands** [comment]
[Website](https://elmanhands.github.io/index.html)
[Repo](https://github.com/Elmanhands/Elmanhands.github.io)

**Comment by aahmadyar123** [comment]
Website: https://aahmadyar123.github.io/
Repo: https://github.com/aahmadyar123/aahmadyar123.github.io

**Comment by meganfung38** [comment]
Repo: https://github.com/meganfung38/meganfung38.github.io.git
Website: https://meganfung38.github.io/

**Comment by syanase-calpoly** [comment]
Website: https://syanase-calpoly.github.io/
Repo: https://github.com/syanase-calpoly/syanase-calpoly.github.io

**Comment by kklave** [comment]
Website: [https://kklave.github.io/](https://kklave.github.io/)
Repo: [https://github.com/kklave/kklave.github.io](https://github.com/kklave/kklave.github.io)

**Comment by aumerov** [comment]
Website: https://aumerov.github.io/
Repository: https://github.com/aumerov/aumerov.github.io
The starter pack was so fun to work on!

**Reply by khoa-l** [reply]
Could you fix the issue and re-submit, please? Just move your files from outside the folder and re-commit!

**Reply by aumerov** [reply]
Fixed the issue!

**Reply by khoa-l** [reply]
Awesome!

**Comment by LambNNot** [comment]
Website: https://lambnnot.github.io/
Repo: https://github.com/LambNNot/LambNNot.github.io
Unfortunately, I couldn't get the image overlays to work, which would've been really cool IMO. Given how much I procrastinated, this turned out a lot better than I expected. (Heavy inspiration from ProjectMili's Key Instrument backgrounds)

**Comment by anvch** [comment]
Website: https://anvch.github.io/
Repository: https://github.com/anvch/anvch.github.io

**Comment by ssponsler** [comment]
Website: ssponsler.github.io
repo: https://github.com/ssponsler/ssponsler.github.io
was supposed to post the reply here last night but i guess i lost connection when I hit "Comment", finished yesterday :)

**Comment by Daniel-Almond1** [comment]
Thank you for considering my website! I learned a lot! 
Website: https://daniel-almond1.github.io/index.html
Repo:  https://github.com/Daniel-Almond1/Daniel-Almond1.github.io

Sorry its a little late. wanted it to be perfect!

**Reply by khoa-l** [reply]
I noticed your image isn't showing up! I think it's because your image path is wrong, instead of an absolute path i.e. `Users\Desktop\etc\IMG_1055.jpeg`, you can use a relative path `IMG_1055.jpeg` since it's in the same folder as all your code!

**Reply by Daniel-Almond1** [reply]
Fixed! Thank you!

**Comment by poornimagodavarthy** [comment]
website: https://poornimagodavarthy.github.io/
repo: https://github.com/poornimagodavarthy/poornimagodavarthy.github.io

**Comment by achan214** [comment]
Website: https://achan214.github.io/
Repo: https://github.com/achan214/achan214.github.io
This was such a fun little project to work on! I learned so much, thank you!

**Comment by ethanoutangoun** [comment]
I already have an existing portfolio hosted on my github pages! I have deployed my website using Vercel, I hope that is okay.

Website: https://ethanoutangoun-website.vercel.app/
Repo: https://github.com/ethanoutangoun/ethanoutangoun-website

**Comment by elainalyons** [comment]
Website: https://elainalyons.github.io/ 
Repo: https://github.com/elainalyons/elainalyons.github.io
This was such a fun project! The starter pack was really easy to follow!

**Comment by IvanA673** [comment]
[Website](https://ivana673.github.io/)
[Repo](https://github.com/IvanA673/IvanA673.github.io)
Thank you for the knowledge on creating a website!

**Comment by Westluu** [comment]
Website: https://westluu.github.io/Hack4Impact-Starter-Pack/
Repo: https://github.com/Westluu/Hack4Impact-Starter-Pack

**Reply by khoa-l** [reply]
Hi Wesley! It seems like your link doesn't work. I can access it through your GitHub, but I think these links are just broken. Would you fix them?

**Reply by Westluu** [reply]
Sorry about that, the links so be working now

**Comment by HenryHorse** [comment]
website: https://henryhorse.github.io/
repo: https://github.com/HenryHorse/HenryHorse.github.io

**Comment by Juan-Gon03** [comment]
web: https://juan-gon03.github.io/Hack4Impact_starter/home.html
repo: https://github.com/Juan-Gon03/Hack4Impact_starter

**Comment by asarelcastellanos** [comment]
website: https://asarelcastellanos.github.io/
repository: https://github.com/asarelcastellanos/asarelcastellanos.github.io

**Comment by kendricktran** [comment]
website: https://kendricktran.github.io/
repo: https://github.com/kendricktran/kendricktran.github.io

**Reply by khoa-l** [reply]
Hi, Kendrick. I noticed your pages don't have the appropriate styling. Would you please fix this and resubmit?

**Reply by kendricktran** [reply]
Fixed! Sorry about that a couple of bugs with my navbar closers and head messed up the css.

**Comment by MelikaDab** [comment]
website: https://MelikaDab.github.io/
repo: https://github.com/MelikaDab/MelikaDab.github.io

**Comment by ArjunR27** [comment]
website: https://https://arjunr27.github.io/
repo: https://github.com/ArjunR27/ArjunR27.github.io

**Reply by khoa-l** [reply]
Hi Arjun! It seems as if your link to your website is formatted improperly and the portfolio has nothing on it? Would you correct those and resubmit?

**Reply by ArjunR27** [reply]
website: https://arjunr27.github.io/
There were accidentally two https:// lol :)

**Comment by cwong239** [comment]
Website: https://cwong239.github.io/
Repo: https://github.com/cwong239/cwong239.github.io

**Comment by Belal-Elshenety** [comment]
Here is my [website](https://belal-elshenety.github.io/)

**Reply by Belal-Elshenety** [reply]
Also this is my repository: [https://github.com/Belal-Elshenety/Belal-Elshenety.github.io](https://github.com/Belal-Elshenety/Belal-Elshenety.github.io)

**Comment by amalekia** [comment]
Hey guys this is my website: [link](https://amalekia.github.io/)
Also my repo: [Repo](https://github.com/amalekia/amalekia.github.io)

I didn't get to do too much to it since I was very loaded this past week and weekend however I do plan on adding onto it from here on out!

**Comment by ryyyvo** [comment]
Final website: https://ryyyvo.github.io/
The repository for it: https://github.com/ryyyvo/ryyyvo.github.io
Thanks!

**Comment by jhueyy** [comment]
Website: https://jhueyy.github.io/
Repository: https://github.com/jhueyy/jhueyy.github.io

**Comment by SumedhaKun** [comment]
Website: https://sumedhakun.github.io/
Repository: https://github.com/SumedhaKun/SumedhaKun.github.io

**Comment by kylelin23** [comment]
Website: https://kylelin23.github.io/kylelin.github.io/
Repository: https://github.com/kylelin23/kylelin.github.io

**Comment by sgill33** [comment]
Website: https://sgill33.github.io/
Repository: https://github.com/sgill33/sgill33.github.io

**Comment by JosephPerez831** [comment]
Website: https://josephperez831.github.io/
Repository: https://github.com/JosephPerez831/JosephPerez831.github.io.git

**Reply by khoa-l** [reply]
Hi Joseph! It seems as if you have your own styling which is great! But, you should add the basic CSS provided! Specifically for the navbar and footer.

**Comment by Castro19** [comment]
Website: https://castro19.github.io/hack4impact/
Repository: https://github.com/Castro19/hack4impact

**Reply by Nathaynn** [reply]
central valley 😎

**Comment by rachanaraju3** [comment]
Website: https://rachanaraju3.github.io/
Repository: https://github.com/rachanaraju3/rachanaraju3.github.io

**Comment by jam-kt** [comment]
Website: https://jam-kt.github.io
Repo: https://github.com/jam-kt/jam-kt.github.io

**Comment by michaelmurrayiv** [comment]
Website: https://michaelmurrayiv.github.io/ 
Repo: https://github.com/michaelmurrayiv/michaelmurrayiv.github.io

**Comment by kaylantran** [comment]
website: https://k4yluh.github.io
repo: https://github.com/k4yluh/k4yluh.github.io

**Comment by clngo** [comment]
Website：https://clngo.github.io/
Repo: https://github.com/clngo/clngo.github.io

**Comment by jojayama** [comment]
Website: https://jojayama.github.io/
Repo: https://github.com/jojayama/jojayama.github.io

**Comment by kbecke05** [comment]
Website: https://kbecke05.github.io/
Repo: https://github.com/kbecke05/kbecke05.github.io

**Reply by khoa-l** [reply]
Hi Kelly! It seems your images are not working on your `index.html` and `portfolio` because your `src=""` is using your computer's absolute path. You can change it to just the name of the image to since it is in the same repository and it should show up!

Next time if you have issues, feel free to make a post in the discussion!

**Comment by tarannssingh** [comment]
Website: https://tarannssingh.github.io/
Repo: https://github.com/tarannssingh/tarannssingh.github.io/tree/main

**Comment by leilat014** [comment]
Website: https://leilat014.github.io/
Repo: https://github.com/leilat014/leilat014.github.io

**Comment by elhagen13** [comment]
Website: https://elhagen13.github.io 
Repo: https://github.com/elhagen13/elhagen13.github.io

**Comment by keilamohan** [comment]
Hello! I plan on continuing to update my website, but here is what I have for the starter pack :)

REPO: https://github.com/keilamohan/keilamohan.github.io
WEBSITE: https://keilamohan.github.io/index.html

**Comment by palldas** [comment]
website: https://palldas.github.io/
repo: https://github.com/palldas/palldas.github.io

**Comment by soramicha** [comment]
Website: https://soramicha.github.io
Repo: https://github.com/soramicha/soramicha.github.io

**Comment by xgarcia17** [comment]
Yay! I've made my website! I plan on updating it over time (like blogging and the resume), but it is ready for submission.
Website: https://xgarcia17.github.io 
Repository: https://github.com/xgarcia17/xgarcia17.github.io

**Comment by sab115** [comment]
Website: https://sab115.github.io/
Repository: https://github.com/sab115/sab115.github.io

**Comment by ajesus7** [comment]
I think I'm satisfied for now!

Website: https://ajesus7.github.io/starter-pack/
Repository: https://github.com/ajesus7/starter-pack

**Comment by joseph-macfarlane** [comment]
Website: https://joseph-macfarlane.github.io/index.html
Repository: https://github.com/joseph-macfarlane/joseph-macfarlane.github.io

**Comment by srishmaulik** [comment]
Made my first Website! I want to keep developing it for later, but I think it is ready for submission
Website- https://srishmaulik.github.io/   
Repository - https://github.com/srishmaulik/srishmaulik.github.io

**Comment by ktaschek** [comment]
Website: https://ktaschek.github.io/
Repo: https://github.com/ktaschek/ktaschek.github.io

**Reply by javalosr2004** [reply]
Very nice website!

**Comment by aryanbaldua** [comment]
website: https://aryanbaldua.github.io/
repo: https://github.com/aryanbaldua/aryanbaldua.github.io#aryans-personal-website

**Comment by ivannghi** [comment]
Website: https://ivannghi-I.github.io/index.html
Repo: https://github.com/ivannghi/ivannghi.github.io

**Reply by javalosr2004** [reply]
Hi Ivan, something went wrong with your github.io page. I have attached a possible fix to this issue: https://github.com/hack4impact-calpoly/starter-pack/discussions/5#discussioncomment-7077783.

**Comment by diegonieves8** [comment]
Website: https://diegonieves8.github.io/
Repo: https://github.com/diegonieves8/diegonieves8.github.io

**Comment by YaseminAkkaya** [comment]
website: https://yaseminakkaya.github.io/
repo: https://github.com/YaseminAkkaya/YaseminAkkaya.github.io.git

**Reply by khoa-l** [reply]
Hi Yasemin!

If you could move your repository outside of the folder and resubmit the files to your repo. It's preventing your website from loading. Please take a look at some other repositories if you need an idea what I mean or reach out if you have questions.

**Reply by YaseminAkkaya** [reply]
Hi Khoa!

Just fixed the problem! Updated links should work now. Thanks for the help :)

Updated Links-
Website: https://yaseminakkaya.github.io/
Repo: https://github.com/YaseminAkkaya/YaseminAkkaya.github.io.git

**Reply by khoa-l** [reply]
Great!

**Comment by sofijadimitrijevic1** [comment]
Website: https://sofijadimitrijevic1.github.io/
Repo: https://github.com/sofijadimitrijevic1/sofijadimitrijevic1.github.io

**Reply by khoa-l** [reply]
Hi!

I think there are some issues with your links! Otherwise, I can access the website by typing them in, but if you could fix the links that'd be amazing!

**Reply by sofijadimitrijevic1** [reply]
Shoot that's so weird, do they work now? I just retyped them

**Reply by khoa-l** [reply]
Yup!

**Comment by alec-cheng3** [comment]
Website: file: https://alec-cheng3.github.io/
Repository: https://github.com/alec-cheng3/Alec-cheng3.github.io

**Comment by stkeopila** [comment]
Website file: https://stkeopila.github.io/
Repository: https://github.com/stkeopila/stkeopila.github.io

**Comment by billymike02** [comment]
Website: https://billymike02.github.io/
Repository: https://github.com/billymike02/billymike02.github.io/tree/main

I made this all in one afternoon (before and after bike night) so be gentle on me

**Reply by ajesus7** [reply]
contact page made me laugh

**Comment by FlowerCake13** [comment]
[Website](https://flowercake13.github.io/hack4impact-starter/)
[Repository](https://github.com/FlowerCake13/hack4impact-starter)

**Comment by sophiapeckner** [comment]
Website: https://sophiapeckner.github.io/
Repository: https://github.com/sophiapeckner/sophiapeckner.github.io

Check these things out:
-Proud of the .onclick() functionality of the images in projects section
-Tried to make my webpage responsive by changing the profile image in the about section based on the viewport size! Maybe try resizing the page:)

Would've loved to have incorporated animations (i.e. make text fly out from the left when scroll) or made the background interesting by putting up an underlying geometric design.

**Comment by Sam2545** [comment]
Website: https://sam2545.github.io/
Repository: https://github.com/Sam2545/Sam2545.github.io 

Hope you like it!

**Comment by 0x10jalencas** [comment]
website: https://0x10jalencas.github.io
repository: [https:/github.com/0x10jalencas/0x10jalencas.github.io](https://github.com/0x10jalencas/0x10jalencas.github.io)

**Comment by dhvani427** [comment]
Website: https://dhvani427.github.io/
Repository: https://github.com/dhvani427/dhvani427.github.io

**Comment by Vishnu-Alachi123** [comment]
Website : https://vishnu-alachi123.github.io/Vishnu-A.github.io/
Repository : https://github.com/Vishnu-Alachi123/Vishnu-A.github.io

**Comment by peterchinh** [comment]
Website: https://peterchinh.github.io/
Repository: https://github.com/peterchinh/peterchinh.github.io

**Comment by HanYu-Wu04** [comment]
Website: https://HanYu-Wu04.github.io/
Repository: https://github.com/HanYu-Wu04/HanYu-Wu04.github.io

**Comment by nriley-14** [comment]
[Website](https://nriley-14.github.io/)
[Repo](https://github.com/nriley-14/nriley-14.github.io)

**Comment by drdonv** [comment]
[Website](https://drdonv.github.io/)
[Repo](https://github.com/drdonv/drdonv.github.io)

**Comment by thienan11** [comment]
[Website](https://thienan11-hack4impact-website.vercel.app)
[Repository](https://github.com/thienan11/thienan11-hack4impact-website) 
I already have a portfolio hosted on GitHub pages so I used Vercel for this. Hope y'all like it!

**Comment by spaykel** [comment]
[HTML Website](https://spaykel.com/hack4impact.github.io/index.html)
[HTML Repository](https://github.com/spaykel/hack4impact.github.io)

Additionally, I have also made another version of my personal portfolio website with React, which is linked here:
https://spaykel.com/
[React Repository](https://github.com/spaykel/spaykel.github.io)

Hope you like it, and I'm interested to see which version people like more!

**Comment by Velevynn** [comment]
[Website](https://velevynn.github.io/)
[Repository](https://github.com/Velevynn/velevynn.github.io)

Decided to add a little bit of theming of it to see how well I could keep a continuous and coherent experience throughout the whole site while having to add little flairs and details. Definitely helped me refresh my CSS, and even found some new attribute and tools to work with!

**Comment by Kyan73** [comment]
Website: https://kyan73.github.io/index.html
Repository https://github.com/Kyan73/Kyan73.github.io.git

**Reply by khoa-l** [reply]
Hi Kyan, it looks like you have no styling on your website?

**Comment by kyramirez** [comment]
website: https://kyramirez.github.io/index.html
repository: https://github.com/kyramirez/kyramirez.github.io

**Reply by khoa-l** [reply]
HI Kyra, it looks like you have no styling on your website?

**Reply by kyramirez** [reply]
Sorry it should work now, thank you!

**Reply by kyramirez** [reply]
file:///Users/kyraramirez/%3Ckyramirez%3E.github.io/index.html
for some reason, this is the link to my website that has been getting updated. but my old link should have style now

**Comment by ishwaryanam** [comment]
website: https://ishwaryanam.github.io/
repo: https://github.com/ishwaryanam/ishwaryanam.github.io

**Comment by idavoong** [comment]
website: https://idavoong.github.io/
repo: https://github.com/idavoong/idavoong.github.io

**Comment by mio-naka** [comment]
website: https://mio-naka.github.io/
repo: https://github.com/mio-naka/mio-naka.github.io

**Comment by gracebw7** [comment]
website: https://gracebw7.github.io/
repo: https://github.com/gracebw7/gracebw7.github.io

**Comment by tammy-si** [comment]
website: [https://tammy-si.github.io/index.html](https://tammy-si.github.io/index.html)
repo: [https://tammy-si.github.io/index.html](https://github.com/tammy-si/tammy-si.github.io )

**Comment by whiniy** [comment]
website: https://whiniy.github.io/
repo: https://github.com/whiniy/whiniy.github.io

**Comment by feliciaj628** [comment]
website: https://feliciaj628.github.io/
repo: https://github.com/feliciaj628/feliciaj628.github.io
thank you! :)

**Comment by Antonio0913** [comment]
website: https://antonio0913.github.io/index.html
repo: https://github.com/Antonio0913/Antonio0913.github.io

**Comment by rcereno** [comment]
website: https://rcereno.github.io/
repo: https://github.com/rcereno/rcereno.github.io
thank you for a great starter pack! I had so much fun and learned a lot! hope you enjoy my website!

**Comment by emily-yarvis** [comment]
Website: https://emily-yarvis.github.io/
Repo: https://github.com/emily-yarvis/emily-yarvis.github.io.git

**Reply by khoa-l** [reply]
Hi Emily! Your photo isn't showing up, it may be because it has a `.JPG` instead of `.jpg`

**Reply by emily-yarvis** [reply]
Hey! Thank you for letting me know I just fixed it!

**Comment by riyabadadare** [comment]
website: https://riyabadadare.github.io/
repo: https://github.com/riyabadadare/riyabadadare.github.io

**Comment by jaylanwong** [comment]
website: https://jaylanwong.github.io/
repo: https://github.com/jaylanwong/jaylanwong.github.io

**Comment by kmanikon** [comment]
website: https://kmanikon.github.io/
repo: https://github.com/kmanikon/kmanikon.github.io

**Comment by jeremiahliao** [comment]
website: https://jeremiahliao.github.io/
repo: https://github.com/jeremiahliao/jeremiahliao.github.io

**Comment by jlabbeCalPoly** [comment]
website: https://jlabbecalpoly.github.io
repo: https://github.com/jlabbeCalPoly/jlabbeCalPoly.github.io

**Comment by seanmnguyen** [comment]
website: https://seanmnguyen.github.io/
repo: https://github.com/seanmnguyen/seanmnguyen.github.io

**Comment by Martin-Wise** [comment]
Website: https://martin-wise.github.io/
Repo: https://github.com/Martin-Wise/Martin-Wise.github.io

**Comment by ishavarrier** [comment]
website link: https://ishavarrier.github.io/-ishavarrier-.github.io/index.html
repository: https://github.com/ishavarrier/-ishavarrier-.github.io

**Comment by Eman-Gon** [comment]
Website link: http://eman-gon.github.io/
repository: [//github.com/Eman-Gon/Eman-Gon.github.io/tree/main/Eman-Gon%20github.io](https://github.com/Eman-Gon/Eman-Gon.github.io)

**Reply by khoa-l** [reply]
Hi Eman, it looks like you need to rename your repository to `Eman-Gon.github.io`. It's missing a period between your name and github. Go to settings in your repository and change it from there!

**Reply by khoa-l** [reply]
Please fix your links in this post!

**Reply by Eman-Gon** [reply]
Thank you! fixed.

**Comment by Bolt259** [comment]
[website](https://Bolt259.github.io)
[repo](https://github.com/Bolt259/Bolt259.github.io)
